src/components/Admin/AdminTwitterAccounts.js

```diff
@@ -1,80 +1,42 @@
-import {DataGrid} from './DataGrid';
+import { DataGrid } from './DataGrid';
 import React from 'react';
 import Fluxxor from 'fluxxor';
 
-const FluxMixin = Fluxxor.FluxMixin(React),
-    StoreWatchMixin = Fluxxor.StoreWatchMixin("AdminStore");
-
-const columns = [
-    {editable: true, 
-     required: true,
-     key: "accountName", 
-     name: "Twitter App Name"},
-    {
-        editable:true,
-        sortable : true,
-        filterable: false,
-        resizable: true,
-        required: true,
-        name: "Consumer Key",
-        key: "consumerKey"
-    },
-    {
-        editable:true,
-        sortable : true,
-        required: true,
-        filterable: false,
-        resizable: true,
-        name: "Consumer Secret",
-        key: "consumerSecret"
-    },
-    {
-        editable:true,
-        sortable : true,
-        required: true,
-        filterable: false,
-        resizable: true,
-        name: "Token",
-        key: "token"
-    },
-    {
-        editable:true,
-        sortable : true,
-        required: true,
-        filterable: false,
-        resizable: true,
-        name: "Token Secret",
-        key: "tokenSecret"
-    }
-];
+const FluxMixin = Fluxxor.FluxMixin(React);
+const StoreWatchMixin = Fluxxor.StoreWatchMixin("AdminStore");
 
 export const AdminTwitterAccounts = React.createClass({
```

Comment:
If you're cracking open this file anyways, consider moving this to an ES6 class instead of using the deprecated createClass. The only thing you'll have to change is to make sure that methods you're passing to sub-components correctly bind the `this` context since you get that for free currently with the createClass call. I like [Approach 5 in this article](https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56).